client/jetpack-connect/test/__snapshots__/signup.js.snap

```diff
@@ -212,8 +212,7 @@ exports[`JetpackSignup should render 1`] = `
                 .
               </p>
               <button
-                class="button signup-form__submit form-button is-primary"
-                locale="en"
```

Comment:
I cannot find any code which depends on this non-standard attribute. I'd expect if it was used it would be `data-attribute`.